src/main/kotlin/build/buf/intellij/manifest/Digest.kt

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package build.buf.intellij.manifest
+
+data class Digest(
+    val digestType: String,
+    val hex: String,
+) {
+    override fun toString(): String {
```

Comment:
Is this used? If it is, maybe instead use a new method?

Reply:
This is just for debugging and to match the Go implementation.